### PR TITLE
feat(sort-by): implement InstantSearch.css

### PR DESCRIPTION
**Pull request by francoischalifour**
## Specs

https://instantsearch-css.netlify.com/widgets/sort-by/

## Stories

https://deploy-preview-3120--algolia-instantsearch.netlify.com/v2/dev-novel/?selectedStory=SortBy.default

### CODE REVIEW

File: src/connectors/sort-by/connectSortBy.js [hunk 1]
```diff
@@ -13,37 +13,37 @@ var customSortBySelector = connectSortBySelector(function render(params, isFirst
...
     [ transformItems ]
   })
 );
-Full documentation available at https://community.algolia.com/instantsearch.js/v2/connectors/connectSortBySelector.html
+Full documentation available at https://community.algolia.com/instantsearch.js/v2/connectors/connectSortBy.html
 `;
 
 /**
- * @typedef {Object} SortBySelectorIndices
+ * @typedef {Object} SortByItems
```
Comment (bobylito) [inline src/connectors/sort-by/connectSortBy.js]: Maybe singular? (not important)

File: src/connectors/sort-by/connectSortBy.js [hunk 2]
```diff
@@ -52,78 +52,75 @@ Full documentation available at https://community.algolia.com/instantsearch.js/v
...
-        const isIndexInList = find(
-          indices,
-          ({ name }) => name === currentIndex
-        );
+        const isIndexInList = find(items, ({ name }) => name === currentIndex);
 
         if (!isIndexInList) {
           throw new Error(
-            `[sortBySelector]: Index ${currentIndex} not present in \`indices\``
+            `[sortBy]: Index ${currentIndex} not present in \`indices\``
```
Comment (bobylito) [inline src/connectors/sort-by/connectSortBy.js]: indices should be replaced by items.

**Comment by francoischalifour** [comment]
Done @bobylito.

**Comment by bobylito** [comment]
I'll let you merge the changes :)

**Comment by francoischalifour** [comment]
The story seems broken [on deploy](https://deploy-preview-3120--algolia-instantsearch.netlify.com/v2/dev-novel/?selectedStory=SortBy.default):

```
Uncaught ReferenceError: You can't access 'instantsearch.widgets' directly from the ES6 build.
Import the widgets this way: 'import {searchBox} from "instantsearch.js/es/widgets"'
    at Function.get (index.js:19)
    at sort-by.stories.js:15
    at wrap-with-hits.js:78
    at e.value (dev-novel.js:170)
    at index.js:55
    at O (mobx.module.js:911)
    at n (mobx.module.js:902)
    at Object.<anonymous> (index.js:42)
    at t (bootstrap 78695c623afd6e3f043e:19)
    at bootstrap 78695c623afd6e3f043e:62
```

On my local machine:

```
ERROR in ./dist-es5-module/src/components/Slider/Slider.js
Module not found: Error: Can't resolve 'react-nouislider' in '/Users/francoischalifour/Development/algolia/instantsearch.js/dist-es5-module/src/components/Slider'
 @ ./dist-es5-module/src/components/Slider/Slider.js 37:23-50
 @ ./dist-es5-module/src/widgets/range-slider/range-slider.js
 @ ./dist-es5-module/src/lib/main.js
 @ ./dist-es5-module/index.js
 @ ./dev/app/builtin/stories/sort-by.stories.js
 @ ./dev/app/builtin/init-stories.js
 @ ./dev/app/index.js
```

I'm not sure to understand.

**Pull request merged**